README.md

```diff
@@ -45,16 +45,14 @@ To stop and delete the database:
 docker-compose down
 ```
 
-## Initializing/Updating the Database
+## Generating Enumerated Types
 
-To apply all unapplied migrations (for an empty database this means all of them) in order, do:
+There are many enumerated types in the database, represented by tables named `e_whatever`. The Scala equivalents are generated *on demand* by queries, then checked into source control like normal source files. This is only needed if you update the contents of an enum in the schema, or add/modify a the generation
+code in the `gen` project. In any case, you can [re]-generate the enumerated types thus:
```

Comment:
Maybe mention that the build will check that the generated files are checked in

Reply:
This check isn't enabled in this repo yet. I'll open an issue.